.ci/build.sh

```diff
@@ -29,7 +29,7 @@ function build() {
 
     if pmd_ci_utils_is_fork_or_pull_request; then
         pmd_ci_log_group_start "Build with mvnw"
-            ./mvnw clean verify --show-version --errors --batch-mode --no-transfer-progress "${PMD_MAVEN_EXTRA_OPTS[@]}"
+            ./mvnw clean install --show-version --errors --batch-mode --no-transfer-progress "${PMD_MAVEN_EXTRA_OPTS[@]}"
```

Comment:
This change is missing on the pmd7 branch, which is probably the reason why we saw the incompatible changes only after it was merged....

Reply:
Indeed, nice find!